packages/strapi-design-system/src/SubNav/SubNavHeader.tsx

```diff
@@ -14,17 +13,39 @@ import { IconButton } from '../IconButton';
 import { Searchbar, SearchForm } from '../Searchbar';
 import { Typography } from '../Typography';
 
+export interface SubNavHeaderProps {
+  as: keyof HTMLElementTagNameMap;
+  id: string;
+  label: string;
+  onChange: ChangeEventHandler<HTMLInputElement>;
```

Comment:
you've set a lot of these props as required, yet you've set a default? I think if they're required, I wouldn't set a default – personally i opt to make them optional, but that's really splitting hairs so it doesn't matter which you prefer.